Tests/Tests/SDWebImageDownloaderTests.m

```diff
@@ -181,9 +181,9 @@ - (void)test11ThatCancelWorks {
 
 - (void)test12ThatWeCanUseAnotherSessionForEachDownloadOperation {
     XCTestExpectation *expectation = [self expectationWithDescription:@"Owned session"];
-    NSURL *imageURL = [NSURL URLWithString:kTestJPEGURL];
```

Comment:
Small question, does this variable naming cause a Xcode warning ? 😕 Anyway it's OK, just a little curious

Reply:
It's not that obvious, but the name was `imageURL` and that was shadowed by a param with the same name in the `addHandlersForProgress` block below it:
```obj-c
SDWebImageDownloaderOperation *operation = [[SDWebImageDownloaderOperation alloc] initWithRequest:request
                                                                                            inSession:nil
                                                                                              options:0];
[operation addHandlersForProgress:^(NSInteger receivedSize, NSInteger expectedSize, NSURL *imageURL) {
} completed:^(UIImage * _Nullable image, NSData * _Nullable data, NSError * _Nullable error, BOOL finished) {
    if (image && data && !error && finished) {
        [expectation fulfill];
    } else {
        XCTFail(@"Something went wrong");
    }
}];
```
All the other changes are similar, there are blocks where the param is shadowed

Reply:
OK.